web-common/src/features/file-explorer/ForceDeleteConfirmation.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  import { Button } from "@rilldata/web-common/components/button";
+  import Dialog from "@rilldata/web-common/components/dialog/Dialog.svelte";
+
+  export let open: boolean;
+  export let onDelete: () => void;
+
+  function handleClose() {
+    open = false;
+  }
+</script>
+
+<Dialog on:close={handleClose} {open}>
+  <div slot="title">
+    Folder is not empty. Are you sure you want to delete everything?
+  </div>
```

Comment:
Can we change the title to: "Are you sure you want to delete everything?"

And add a `body` section: "This folder is not empty. All contained items will be deleted."